runtime/android/core/strings/xwalk_app_strings.grd

```diff
@@ -2,62 +2,59 @@
-        Decompressing Crosswalk Library
+        Decompressing Crosswalk Project runtime...
       </message>
       <message desc="Download Progress Message [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_PROGRESS_MESSAGE">
-        Downloading Crosswalk
-      </message>
-      <message desc="Download Failed Title [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_TITLE">
-        Crosswalk Download Failed
+        Downloading Crosswalk Project from remote host...
       </message>
       <message desc="Download Failed Message [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_MESSAGE">
-        Please retry or cancel
+        Download failed. A network error occurred.
       </message>
       <message desc="Download Failed Not Found [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_DEVICE_NOT_FOUND">
-        Storage device not found
+        Download failed. Storage device not found.
       </message>
       <message desc="Download Failed Insufficient space [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_INSUFFICIENT_SPACE">
-        Insufficient storage space
+        Download failed. Insufficient disk space.
       </message>
       <message desc="Download Failed Time Out [CHAR-LIMIT=100]" name="IDS_DOWNLOAD_FAILED_TIME_OUT">
-        Timed out
-      </message>
-      <message desc="Market Open Failed Title [CHAR-LIMIT=100]" name="IDS_MARKET_OPEN_FAILED_TITLE">
-        Market Open Failed
+        Download failed. The network connection timed out.
       </message>
       <message desc="Market Open Failed Message [CHAR-LIMIT=100]" name="IDS_MARKET_OPEN_FAILED_MESSAGE">
-        Please set up your market
+        Failed to open the app store
       </message>
       <message desc="Startup Not Found Title [CHAR-LIMIT=100]" name="IDS_STARTUP_NOT_FOUND_TITLE">
-        Crosswalk Not Found
+        Crosswalk Project Not Found
       </message>
       <message desc="Startup Not Found Message [CHAR-LIMIT=100]" name="IDS_STARTUP_NOT_FOUND_MESSAGE">
-        Please get Crosswalk Runtime Library
+        This application requires the Crosswalk Project runtime to work. Select "Continue" to download it.
       </message>
       <message desc="Startup Architecture Mismatched Title [CHAR-LIMIT=100]" name="IDS_STARTUP_ARCHITECTURE_MISMATCH_TITLE">
-        Mismatch of CPU Architecture for Crosswalk
+        Mismatch of CPU Architecture for Crosswalk Project
       </message>
       <message desc="Startup Architecture Mismatched Message [CHAR-LIMIT=100]" name="IDS_STARTUP_ARCHITECTURE_MISMATCH_MESSAGE">
-        Please get Crosswalk Runtime Library
+        The Crosswalk Project must be updated to match the CPU Architecture of the device. Select "Continue" to download it.
       </message>
       <message desc="Startup Signature Check Error Title [CHAR-LIMIT=100]" name="IDS_STARTUP_SIGNATURE_CHECK_ERROR_TITLE">
-        Crosswalk Signature Verification Failed
+        Crosswalk Project Signature Verification Failed
       </message>
```

Comment:
Is "Signature Verification" a program?  Why is it capitalized?  What about:  "The Crosswalk Project signature verification failed."   Also, is there something they can do, like re-download it?  If so maybe add "Please re-install the Crosswalk Project runtime."  (just a guess)